middlewares/redirect_home_unless_have_game.rb

```diff
@@ -0,0 +1,19 @@
+class RedirectHomeUnlessHaveGame < BaseMiddleware
+  def initialize(app)
+    super()
```

Comment:
linter forces me: RuboCop: Call `super` to initialize state of the parent class. [Lint/MissingSuper]. But i moved this constructor to base class to avoid duplication.